examples/handlebars/README.md

```diff
@@ -0,0 +1,33 @@
+# Fractal Handlebars example
+
+This is an example instance of Fractal using the default Handlebars adapter.
+
+## Setup
+
+[Follow these instructions](https://github.com/frctl/fractal#development) to setup the development environement, then run:
+
+```bash
+npm start
```

Comment:
blabbed about this under the Mandelbrot PR, but we should somehow agree on a development process and figure out what the best way to get started is when getting the repo for the first time. Relevant point here is that before running the example instance, you need to build/start mandelbrot.

Reply:
Cross-posting, since it's a bit relevant under both PRs:
> Found that running npx lerna run --scope=@frctl/handlebars-example --include-dependencies --parallel start for the example project will build mandelbrot (and any other dependencies that have a start script) too.